### PR TITLE
fix: Update labeler & drafter workflow labels config to new labels 

### LINKED ISSUE #343: Missing labels and labeler CI inclusion of new workspace member crates.
We should add labels and modify the labeler workflow in order to be able to mark every PR and obtain release reports much better and for each workspace member.

### PR DESCRIPTION
**Pull request by CPerezz**
Resolves: https://github.com/appliedzkp/zkevm-circuits/issues/343

### CODE REVIEW

**Comment by CPerezz** [comment]
This is weird. The workflow succeeds while the labels are not added. I wonder what can be happening. Or if it's the expected behaviour. 

As what I was expecting was to see the `CI` label added.

File: .github/labeler.yml [hunk 1]
```diff
@@ -1,10 +1,35 @@
-# Add 'zkevm-circuits' label to any change to files within the zkevm-circuits dir EXCEPT for the evm_circuit sub-folder
-zkevm-circuits:
-  - any: ['zkevm-circuits/src/**/*', '!zkevm-circuits/src/evm_circuit/**/*']
-# Add 'opcodes' to any changes within 'evm_circuit' folder or any subfolders
-opcode:
-  - zkevm-circuits/src/evm_circuit/**/*
-bus-mapping:
+# Add CI label for any CI changes
+CI:
+  - .github/**/*
```
Comment (ChihChengLiang) [inline .github/labeler.yml:3]: Maybe we should quote this, so yaml doesn't interpret the "." in a strange way? 
```suggestion
  - ".github/**/*"
```

Comment (CPerezz) [reply .github/labeler.yml]: Was a good try. But not working neither.

I'm really not following what's going on.. Will take a deeper look

**Comment by ChihChengLiang** [comment]
Hypothesis 2: The labeler runs without `- uses: actions/checkout@v2`, so it still uses the configs on 'main.' It would take effect when merged.

**Comment by CPerezz** [comment]
> Hypothesis 2: The labeler runs without `- uses: actions/checkout@v2`, so it still uses the configs on 'main.' It would take effect when merged.

Should we try?? In theory, with this config:
```yaml
on:
  - pull_request_target
```
It should trigger here too. I'll try to make a commit that touches some other part of the repo to see if it's only the CI label that is broken.

**Review by ChihChengLiang — APPROVED** [review]
Looks good after a rebase.

**Comment by ed255** [comment]
> > Hypothesis 2: The labeler runs without `- uses: actions/checkout@v2`, so it still uses the configs on 'main.' It would take effect when merged.
> 
> Should we try?? In theory, with this config:
> 
> ```yaml
> on:
>   - pull_request_target
> ```
> 
> It should trigger here too. I'll try to make a commit that touches some other part of the repo to see if it's only the CI label that is broken.

According to github docs https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target when using `pull_request_target` the github action uses the code from main, not from the PR:
> This event runs in the context of the base of the pull request, rather than in the context of the merge commit, as the pull_request event does.
So I think it was expected that this change only would be in effect when merged, and not in this PR.

**Pull request merged**